### ISSUE TITLE
Notebooks  only have instructions for CPD

### ISSUE THREAD

**Issue by natashadsilva** [IBMStreams/sample.starter_notebooks#65]
It is okay if the notebooks only have instructions for running in CPD. The problem is that is not clear that _it will not work as-is_ outside of CPD.  Rolling average sample has example of instructions on how to run outside CPD, but maybe adding a link to complete the 3 steps on this page is enough: https://ibmstreams.github.io/streamsx.documentation/docs/python/1.6/python-appapi-devguide-2/

**Comment by natashadsilva** [comment]
@markheger

**Comment by markheger** [comment]
Yes, notebooks have been designed for Streams in Cloud Pak for Data since ordered for this only.

Adding a link as your proposed is fine for those notebooks that can run outside CPD.

I guess that WML, PMML and EventStore are currently the notebooks that require CPD, all others should mostly be able to run in other environments as well, but requires test effort.

The major problem is that the environments are so different, toolkit versions are very different (for example Streaming Analytics service has very old toolkits and some are missing).
Streaming Analytics service has very old REST API, for example managing "application configuration" is not possible via REST.

**Comment by ghost** [comment]
We could add instructions on how to use the notebooks outside of a CP4D together with Streams in a CP4D, i.e. connect from remote with a CP4D. But does this make sense at all?

**Comment by natashadsilva** [comment]
@markheger  I agree the priority is CPD.  However now I am seeing people get confused because the old Watson Studio is now CPD as  a Service, so they have the same name!!  The problem is not that there are limitations, the problem is that we did not state them or state them clearly, so people get confused as to why it doesn't work.

For the notebooks that require CPD or any other specific API at a specific function, it is enough to state that in the introduction that this notebook is for CPD version X and will not work in CPDaaS and above and requires package xyz. 

We cannot provide instructions for every scenario in the notebook. That is what the documentation is for.

I think it is sufficient if we update the notebooks to state clearly that 
- this requires a CPD project
- you must add a different set of code snippets it you are using a different environment e.g CPDaaS, vanilla Jupyter.
